.solhint.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@AbhinavMir let's add all the recommended rules:

https://protofire.github.io/solhint/docs/rules.html#best-practise-rules